MegaSolution/Sequance/Inputer.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+
+using System;
+
+namespace Sequence
+{
+    class InputClass
+    {
+        const string INSTRUCTIONS1LINE = "Enter a sequance of nubers. ";
+        const string INSTRUCTIONS2LINE = "Use Space to devide numbers. Use Enter to finish input.";
+        public int[] EnterSequense()
```

Comment:
Use at least two methods, one is to input array of strings, other- to transform it to ints, single responsibility